msteams-platform/messaging-extensions/build-bot-based-plugin.md

```diff
@@ -0,0 +1,242 @@
+
+1. From the compose message area, send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
+
+   :::image type="content" source="../assets/images/Copilot/bot-based-VSC-trigger-plugin.png" alt-text="Screenshot shows the plugin prompt and the response from copilot.":::
+
+> [!NOTE]
+> This prompt may not always make Copilot include a response from your message extension. If it happens, try some other prompts or leave feedback to us by thumbing down the Copilot response and leave a message tagged with [MessageExtension].
+
+# [Toolkit CLI](#tab/toolkit-cli)
+
+1. Go to **Command Prompt**.
+
+1. Enter the following command:
+
+   ```
+   npm install -g @microsoft/teamsfx-cli@2.0.3-beta.2023101103.0
+   ```
+
+1. Type `teamsfx new` in the terminal
+
+1. Select **Message Extension**.
+
+   :::image type="content" source="../assets/images/Copilot/api-based-me-CLI-new-project-me.png" alt-text="Screenshot shows Teams capabilities as options in the CLI interface.":::
+
+1. Select **Custom Search Results**.
+1. Select **Start with a Bot**.
+
+   :::image type="content" source="../assets/images/Copilot/bot-based-CLI-new-project.png" alt-text="Screenshot shows the message extension, custom search results and start with a bot options selected in the CLI window.":::
+
+1. Enter the location for your project and select **Enter**.
+
+1. Enter the name of your application and select **Enter**.
+
+   Teams Toolkit will scaffold and create a project in the specified location.
+
+1. Go to the folder path where your project is created and enter the following command to provision your app in Azure:
+
+   ```teamsfx provision --env dev```
+
+   Teams Toolkit CLI opens a browser window and requests you to sign in to your Microsoft Account.
+
+1. Sign in to your Microsoft account. Teams Toolkit CLI will execute validation and provisions your app on Azure.
+
+   :::image type="content" source="../assets/images/Copilot/api-based-CLI-provision-me.png" alt-text="Screenshot shows the sign in request and the provision stages in the command prompt window.":::
+
+1. From the list, select a subscription.
+
+   :::image type="content" source="../assets/images/Copilot/bot-based-CLI-resource-group.png" alt-text="Screenshot shows the login to Azure and select a subscription option in the CLI window.":::
+
+1. From the list, select a resource group.
+1. For Cost may incur according to the usage. Do you want to provision resources in dev environment using accounts listed above?, Enter Y.
+
+   :::image type="content" source="../assets/images/Copilot/bot-based-CLI-provisoin-preview.png" alt-text="Screenshot shows the Do you want to provision resources in dev environment using accounts listed above? option in CLI window.":::
+
+   Teams Toolkit will validate your app and provision your app on Azure.
```

Comment:
Should this be "validate your app *manifest*"? To not be mistaken for Teams store validation

Reply:
Updated.